app/src/main/java/com/duckduckgo/app/browser/BrowserViewModel.kt

```diff
@@ -322,14 +328,33 @@ class BrowserViewModel(
     fun addBookmark(title: String, url: String) {
         bookmarksDao.insert(BookmarkEntity(title = title, url = url))
     }
-    
+
     private fun openUrl(url: String) {
         command.value = Navigate(url)
     }
 
     fun onUserSelectedToEditQuery(query: String) {
         viewState.value = currentViewState().copy(isEditing = false, showAutoCompleteSuggestions = false, omnibarText = query)
     }
+
+    fun userLongPressedInWebView(target: WebView.HitTestResult, menu: ContextMenu) {
+        Timber.i("Long pressed on %s, (extra=%s)", target.type, target.extra)
```

Comment:
Can we use kotlin style strings? It is the project standard and I'd rather not mix styles. I know it's slightly less efficient but on the upside they are more readable and if efficiency becomes an issue we can fix it with some proguard config.

Reply:
I've removed other instances of the _Timber style_ in favour of the _Kotlin style_ too.